### PR TITLE
feat: Make heading links work (#20)

### PR DESCRIPTION
**Pull request by andrew-polk**
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/docu-notion/22)

### CODE REVIEW

**Review by gmartin7 — APPROVED** [review]

File: src/links.ts [hunk 1]
```diff
@@ -8,14 +8,18 @@ export function convertInternalLinks(
...
-    if (p) {
-      verbose(
-        `Converting Link ${url} --> ${layoutStrategy.getLinkPathForPage(p)}`
-      );
-      return layoutStrategy.getLinkPathForPage(p);
+    if (page) {
+      let convertedLink = layoutStrategy.getLinkPathForPage(page);
+
+      // Include the fragment (# and after) if it exists
+      const { fragmentId } = parseLinkId(url);
```
Comment (gmartin7) [inline src/links.ts:18]: I guess you don't need `baseLinkId` here too, just to keep the destructuring happy?

**Review by gmartin7 — APPROVED** [review]
One question, probably fine.

**Review by andrew-polk — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/docu-notion/22)* status: 0 of 5 files reviewed, 1 unresolved discussion (waiting on @gmartin7)

___
*[`src/links.ts` line 18 at r1](https://reviewable.io/reviews/sillsdev/docu-notion/22#-NIDhGqaH9-HCuWKFWu6-r1-18:-NIDizhz0FJ3mFWBv9HE:b-83og3x) ([raw file](https://github.com/sillsdev/docu-notion/blob/088e6a65dac428c5067438cc8608776c11283624/src/links.ts#L18)):*
<details><summary><i>Previously, gmartin7 (Gordon Martin) wrote…</i></summary><blockquote>

I guess you don't need `baseLinkId` here too, just to keep the destructuring happy?
</blockquote></details>

Nope.

**Review by gmartin7 — COMMENTED** [review]
:lgtm:

> Reviewed 5 of 5 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/docu-notion/22#-:-NIDj6KS1ZcV207fobcE:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**